simplq/src/styles/homePage.module.scss

```diff
@@ -51,31 +51,42 @@
   }
 }
 
-.benefits-info {
-  @include background-color();
-  @include text-color();
-  @include center-vertically();
-  @include center-horizontally();
-  padding-bottom: 5rem;
-  h2 {
+.section {
+  padding: 5rem 10rem;
+  @media screen and (max-width: 550px) {
+    padding: 2rem;
+  }
+  & > h2 {
     @include standard-header();
     font-family: $header-font;
     font-size: 2rem;
   }
-  .benefits-container {
+  & > .container {
+    margin: 4rem 0;
+    @media screen and (max-width: 550px) {
+      margin: 2rem;
+    }
```

Comment:
I've been very generous with paddings and margins, but reducing them on smaller screens.